profiles/common/themes/ec_resp/css/interinstituinal_less/eu_resp.less

```diff
@@ -0,0 +1,107 @@
+@import "../../bootstrap/less/pagination.less";
+@import "../../bootstrap/less/pager.less";
+@import "../../bootstrap/less/labels.less";
+@import "../../bootstrap/less/badges.less";
+@import "../../bootstrap/less/jumbotron.less";
+@import "../../bootstrap/less/thumbnails.less";
+@import "../../bootstrap/less/alerts.less";
+@import "../../bootstrap/less/progress-bars.less";
+@import "../../bootstrap/less/media.less";
+@import "../../bootstrap/less/list-group.less";
+@import "../../bootstrap/less/panels.less";
+@import "../../bootstrap/less/wells.less";
+@import "../../bootstrap/less/close.less";
+
+// Components w/ JavaScript
+@import "../../bootstrap/less/modals.less";
+@import "../../bootstrap/less/tooltip.less";
+@import "../../bootstrap/less/popovers.less";
+@import "../../bootstrap/less/carousel.less";
+
+// Utility classes
+@import "../../bootstrap/less/utilities.less";
+@import "../../bootstrap/less/responsive-utilities.less";
+
+/*! 
+ * Ec resp 
+ */
+//import page styles
+@import "../less/style.less"; // global look and feel
+
+@import "../less/header.less"; // banner
+@import "../less/node.less"; // nodes
+@import "../less/menu.less"; // navigation and action menus
+@import "../less/sidebar.less"; // left and right sidebars
+@import "../less/comment.less"; // comment form and comments
+@import "../less/search.less"; // search form and results
+@import "../less/user.less"; // user login and profile
+@import "../less/form.less"; // form elements
+@import "../less/workbench.less"; // workbench
+@import "../less/quicktab.less"; // quicktabs
+@import "../less/feature-faq.less"; // feature faq
+@import "../less/feature-news.less"; // feature news
+@import "../less/feature-communities.less"; // feature communities
+@import "../less/feature-events.less"; // feature events
+@import "../less/feature-ideas.less"; // feature ideas
+@import "../less/feature-media-galleries.less"; // feature media galleries
+@import "../less/feature-fat-footer.less"; // feature fat footer
+@import "../less/feature-splash.less"; // feature splash screen
+@import "../less/feature-wiki.less"; // feature wiki
+@import "../less/feature-blog.less"; // feature blog
+@import "../less/feature-links.less"; // feature links
+@import "../less/feature-e-library.less"; // feature e-library
+@import "../less/feature-content-slider.less"; // feature content slider
+@import "../less/feature-notifications.less"; // feature notifications
+@import "../less/feature-social-bookmark.less"; // feature social bookmark
+@import "../less/feature-forum.less"; // feature forum
+@import "../less/feature-megamenu.less"; // feature mega menu
+@import "../less/feature-set.less"; // feature set
+@import "../less/feature-media.less"; // media related
+@import "../less/feature-profiles.less"; // feature extended profiles
+
+@import "../less/hack-bootstrap.less"; // hack bootstrap
+
+/*!
+ * Specific styles for interinstitutional theme
+ */
+@import "hack-style.less"; // hack style
```

Comment:
Use ``../less/interinstituinal_less/*`` to include interinstitutional less files